src/fee.rs

```diff
@@ -0,0 +1,66 @@
+/// Function for computing a fee.
+
+#[cfg(any(test, feature = "rand"))]
+use crate::errors::LibError;
+
+/// Get fee in vbytes
+#[cfg(any(test, feature = "rand"))]
+#[allow(arithmetic_overflow)]
+pub fn get_fee_vbytes(fee_rate: u64, weight: u64) -> Result<u64, LibError> {
+    if fee_rate == 0 {
+        Err(LibError::FeeRateSize)
+    } else if weight == 0 {
+        Err(LibError::WeightSize)
+    } else {
+        let vbytes = weight / 4;
+        let fee = fee_rate.wrapping_mul(vbytes);
```

Comment:
why don't you just use `checked_mul`?

Reply:
`checked_mul` is part of the `num` crate: https://docs.rs/num/latest/num/trait.CheckedMul.html.  As you said before, it's best to minimize external dependencies.  Is there a different `checked_mul` that I don't know about?

Reply:
https://doc.rust-lang.org/std/primitive.u64.html#method.checked_mul

Reply:
thanks, done.